.rubocop.yml

```diff
@@ -1,12 +1,25 @@
 AllCops:
-  TargetRubyVersion: 2.4
+  NewCops: enable
 
 Metrics/AbcSize:
   Max: 35
 
+Metrics/BlockLength:
+  CountAsOne:
+    - array
+    - hash
+    - heredoc
```

Comment:
I didn't know you could do this; v useful

Reply:
Agreed. I cribbed this from umts/umaps-rails#1521